### DISCUSSION TITLE
searchEngineJump-NextStage 规则转换

### DISCUSSION THREAD

**Discussion by hoothin** [hoothin/SearchJumper#66] [Search engines]
by ted423
<pre>
[
    {
        "type": "网页",
        "icon": "search",
        "sites": [
            {
                "name": "Google",
                "url": "https://www.google.com/search?q=%s&safe=off"
            },
            {
                "name": "Google.hk",
                "url": "https://www.google.com.hk/search?q=%s&safe=off"
            },
            {
                "name": "Google.co.jp",
                "url": "https://www.google.co.jp/search?q=%s&safe=off"
            },
            {
                "name": "百度",
                "url": "https://www.baidu.com/s?wd=%s"
            },
            {
                "name": "360",
                "url": "https://www.so.com/s?q=%s"
            },
            {
                "name": "Bing(CN)",
                "url": "https://cn.bing.com/search?q=%s"
            },
            {
                "name": "Bing(Global)",
                "url": "https://global.bing.com/search?q=%s&setmkt=en-us&setlang=en-us"
            },
            {
                "name": "搜狗",
                "url": "https://www.sogou.com/web?query=%s"
            },
            {
                "name": "DuckDuckGo",
                "url": "https://duckduckgo.com/?q=%s"
            },
            {
                "name": "Yahoo",
                "url": "https://search.yahoo.com/search?p=%s"
            },
            {
                "name": "Yahoo(tw)",
                "url": "https://tw.search.yahoo.com/search?p=%s"
            },
            {
                "name": "Yahoo.co.jp",
                "url": "https://search.yahoo.co.jp/search?p=%s&aq=-1&x=wrt"
            },
            {
                "name": "Startpage",
                "url": "https://www.startpage.com/do/search?q=%s"
            }
        ]
    },
    {
        "type": "资料",
        "icon": "book-open-reader",
        "sites": [
            {
                "name": "中文维基",
                "url": "https://zh.wikipedia.org/w/index.php?search=%s&button=&title=Special%3ASearch"
            },
            {
                "name": "英文维基",
                "url": "https://en.wikipedia.org/w/index.php?search=%s&button=&title=Special%3ASearch"
            },
            {
                "name": "日本维基",
                "url": "https://ja.wikipedia.org/w/index.php?search=%s&button=&title=Special%3ASearch"
            },
            {
                "name": "百度百科",
                "url": "https://baike.baidu.com/searchword/?word=%s"
            },
            {
                "name": "Internet Archive",
                "url": "https://archive.org/search.php?query=%s"
            },
            {
                "name": "Scholar",
                "url": "http://scholar.google.com/scholar?hl=zh-CN&q=%s&btnG=&lr="
            },
            {
                "name": "百度学术",
                "url": "https://xueshu.baidu.com/s?wd=%s"
            },
            {
                "name": "知乎",
                "url": "https://www.zhihu.com/search?q=%s"
            },
            {
                "name": "萌娘百科",
                "url": "http://zh.moegirl.org/index.php?search=%s"
            },
            {
                "name": "Google Book",
                "url": "https://www.google.com/search?q=%s&btnG=搜索图书&tbm=bks&tbo=1&hl=zh-CN&gws_rd=ssl"
            },
            {
                "name": "中国知网",
                "url": "https://kns.cnki.net/KNS8/DefaultResult/Index?dbcode=CFLS&kw=%s"
            },
            {
                "name": "爱学术",
                "url": "https://www.ixueshu.com/search/index.html?search_type=&q=%s"
            },
            {
                "name": "维普",
                "url": "http://lib.cqvip.com/Qikan/Search/Index?from=Qikan_Search_Index/:p{key=u%3D%s&indexKey=%s}"
            },
            {
                "name": "stackoverflow",
                "url": "https://stackoverflow.com/search?q=%s"
            },
            {
                "name": "MDN",
                "url": "https://developer.mozilla.org/en-US/search?q=%s"
            },
            {
                "name": "github",
                "url": "https://github.com/search?q=%s"
            },
            {
                "name": "krugle",
                "url": "http://opensearch.krugle.org/document/search/#query=%s"
            },
            {
                "name": "npm",
                "url": "https://www.npmjs.org/search?q=%s"
            }
        ]
    },
    {
        "type": "地图",
        "icon": "map",
        "sites": [
            {
                "name": "百度",
                "url": "http://map.baidu.com/?newmap=1&s=s%26wd%3D%s"
            },
            {
                "name": "Google",
                "url": "https://www.google.com/maps/search/%s/"
            },
            {
                "name": "Bing",
                "url": "https://www.bing.com/ditu/?q=%s"
            },
            {
                "name": "搜狗",
                "url": "https://map.sogou.com/#lq=%s"
            }
        ]
    },
    {
        "type": "音乐",
        "icon": "music",
        "sites": [
            {
                "name": "千千",
                "url": "https://music.taihe.com/search?word=%s"
            },
            {
                "name": "QQ",
                "url": "https://y.qq.com/portal/search.html#page=1&searchid=1&t=song&w=%s"
            },
            {
                "name": "一听",
                "url": "https://so.1ting.com/song?q=%s"
            },
            {
                "name": "虾米",
                "url": "http://www.xiami.com/search?key=%s"
            },
            {
                "name": "piapro",
                "url": "https://piapro.jp/search/?view=audio&keyword=%s"
            },
            {
                "name": "QQ歌词",
                "url": "https://y.qq.com/portal/search.html#page=1&searchid=1&t=lyric&w=%s"
            },
            {
                "name": "一听歌词",
                "url": "https://so.1ting.com/lyric?q=%s"
            },
            {
                "name": "歌词迷",
                "url": "http://gecimi.com/search/?keyword=%s"
            }
        ]
    },
    {
        "type": "图片",
        "icon": "image",
        "sites": [
            {
                "name": "百度",
                "url": "http://image.baidu.com/search/index?tn=baiduimage&word=%s"
            },
            {
                "name": "Google",
                "url": "https://www.google.com.hk/search?tbm=isch&q=%s"
            },
            {
                "name": "花瓣",
                "url": "https://huaban.com/search/?q=%s"
            },
            {
                "name": "Picsearch",
                "url": "http://cn.picsearch.com/index.cgi?q=%s"
            },
            {
                "name": "Flickr",
                "url": "https://www.flickr.com/search/?w=all&q=%s"
            },
            {
                "name": "Pixiv",
                "url": "http://www.pixiv.net/search.php?s_mode=s_tag&word=%s"
            },
            {
                "name": "dA",
                "url": "https://www.deviantart.com/browse/all/?q=%s"
            }
        ]
    },
    {
        "type": "下载",
        "icon": "download",
        "sites": [
            {
                "name": "BTDigg",
                "url": "https://btdig.com/search?q=%s"
            },
            {
                "name": "idope",
                "url": "https://idope.se/torrent-list/%s"
            },
            {
                "name": "TK",
                "url": "https://www.torrentkitty.tv/search/%s"
            },
            {
                "name": "limetorrents.co",
                "url": "https://www.limetorrents.co/search/all/%s"
            },
            {
                "name": "limetorrents.asia",
                "url": "https://www.limetorrents.asia/search/all/%s"
            },
            {
                "name": "limetorrents.pro",
                "url": "https://www.limetor.pro/search/all/%s"
            },
            {
                "name": "xiaohx",
                "url": "https://v1.xiaohx.org/search?key=%s"
            },
            {
                "name": "cdsoso",
                "url": "https://www.cdsoso.cc/searches-%s-hot-1-null.html"
            },
            {
                "name": "zimuku",
                "url": "http://zimuku.org/search?q=%s"
            },
            {
                "name": "射手网(伪)",
                "url": "https://assrt.net/sub/?searchword=%s"
            },
            {
                "name": "sub HD",
                "url": "https://subhd.la/search/%s"
            },
            {
                "name": "Subscene",
                "url": "https://subscene.com/subtitles/searchbytitle:p{query=%s}"
            },
            {
                "name": "R3SUB",
                "url": "https://r3sub.com/search.php?s=%s"
            }
        ]
    },
    {
        "type": "影视",
        "icon": "video",
        "sites": [
            {
                "name": "torrentfunk2",
                "url": "https://www.torrentfunk2.com/all/torrents/%s.html?v=&smi=&sma=&i=250"
            },
            {
                "name": "thepiratebay",
                "url": "https://thepiratebay.org/search/%s"
            },
            {
                "name": "zooqle",
                "url": "https://zooqle.com/search?q=%s"
            },
            {
                "name": "pianyuan",
                "url": "http://www.pianyuan.la/search?q=%s"
            },
            {
                "name": "Torrentz2",
                "url": "https://www.torrentz2.xyz/?q=%s"
            },
            {
                "name": "Torrentz2k",
                "url": "https://torrentz2k.xyz/search/:p{q=%s&category=all}"
            },
            {
                "name": "1337x.to",
                "url": "https://www.1377x.to/search/%s/1/"
            },
            {
                "name": "Torlock",
                "url": "https://www.torlock2.com/all/torrents/%s.html"
            },
            {
                "name": "YourBT",
                "url": "https://yourbittorrent2.com/?q=%s"
            },
            {
                "name": "TD",
                "url": "https://www.torrentdownloads.me/search/?search=%s"
            },
            {
                "name": "xiaohx",
                "url": "https://www.xiaohx.org/search?key=%s"
            },
            {
                "name": "rarbgprx",
                "url": "https://rarbgprx.org/torrents.php?search=%s"
            },
            {
                "name": "rarbgproxy",
                "url": "https://rarbgproxy.org/torrents.php?search=%s"
            },
            {
                "name": "kickass(镜像1)",
                "url": "https://kat.sx/search.php?q=%s"
            },
            {
                "name": "kickasstorrents",
                "url": "https://kickasstorrents.to/usearch/%s"
            },
            {
                "name": "kickass1.to",
                "url": "https://kickass1.to/usearch/%s/"
            },
            {
                "name": "kat.am",
                "url": "https://kat.am/usearch/%s/"
            },
            {
                "name": "kickasstorrent.cr",
                "url": "https://kickasstorrent.cr/usearch/%s/"
            }
        ]
    },
    {
        "type": "ACG",
        "icon": "gamepad",
        "sites": [
            {
                "name": "nyaa.si",
                "url": "https://nyaa.si/?f=0&c=0_0&q=%s"
            },
            {
                "name": "dmhy",
                "url": "https://share.dmhy.org/topics/list?keyword=%s"
            },
            {
                "name": "Tokyotosho",
                "url": "https://www.tokyotosho.info/search.php?terms=%s"
            },
            {
                "name": "Mikan",
                "url": "http://mikanani.me/Home/Search?searchstr=%s"
            },
            {
                "name": "＊MioBT＊",
                "url": "http://www.miobt.com/search.php?keyword=%s"
            },
            {
                "name": "shana project",
                "url": "https://www.shanaproject.com/search/?title=%s"
            },
            {
                "name": "简单动漫",
                "url": "https://www.36dm.com/search.php?keyword=%s"
            },
            {
                "name": "KOTOMI RSS",
                "url": "https://moe4sale.in/?kw=%s"
            },
            {
                "name": "ACG狗狗",
                "url": "http://bt.acg.gg/search.php?keyword=%s"
            },
            {
                "name": "ACG.RIP",
                "url": "https://acg.rip/?term=%s"
            },
            {
                "name": "AcgnX",
                "url": "https://www.acgnx.se/search.php?sort_id=0&keyword=%s"
            },
            {
                "name": "末日動漫",
                "url": "https://share.acgnx.se/search.php?sort_id=0&keyword=%s"
            }
        ]
    },
    {
        "type": "Sukebei",
        "icon": "file",
        "sites": [
            {
                "name": "nyaa.si",
                "url": "https://sukebei.nyaa.si/?f=0&c=0_0&q=%s"
            },
            {
                "name": "Glodls",
                "url": "https://glodls.to/search_results.php?search=%s"
            },
            {
                "name": "AniDex",
                "url": "https://anidex.info/?q=%s"
            },
            {
                "name": "AcgnX Ero",
                "url": "https://www.anix.moe/search.php?sort_id=0&keyword=%s"
            },
            {
                "name": "AniRena",
                "url": "https://www.anirena.com/?s=%s"
            },
            {
                "name": "GGBases",
                "url": "http://www.ggbases.com/search.so?title=%s"
            }
        ]
    },
    {
        "type": "Etc",
        "icon": "suitcase",
        "sites": [
            {
                "name": "邮编库",
                "url": "http://www.youbianku.com/%s"
            },
            {
                "name": "AMO",
                "url": "https://addons.mozilla.org/zh-CN/firefox/search/?q=%s"
            },
            {
                "name": "汉典(字)",
                "url": "http://www.zdic.net/sousuo/?q=%s&tp=tp1"
            },
            {
                "name": "汉典(词)",
                "url": "http://www.zdic.net/sousuo/?q=%s&tp=tp3"
            }
        ]
    }
]
</pre>

**Comment by Yankii001** [comment]
_[[audiomack.page.link.docx](https://github.com/user-attachments/files/24477310/audiomack.page.link.docx)]()_****

**Reply by Yankii001** [reply]
@RomanAleynikov
